src/algorithms.jl

```diff
@@ -4086,6 +4086,240 @@ function Base.show(io::IO, alg::OwrenZen5)
           ", thread = ", alg.thread, ")")
 end
 
+
+"""
+    Alshina2(; stage_limiter! = OrdinaryDiffEq.trivial_limiter!,
+             step_limiter! = OrdinaryDiffEq.trivial_limiter!,
+             thread = OrdinaryDiffEq.False())
+
+2nd order, 2-stage Explicit Runge-Kutta Method with optimal parameters.
+
+Like SSPRK methods, this method also takes optional arguments `stage_limiter!`
+and `step_limiter!`, where `stage_limiter!` and `step_limiter!` are functions
+of the form `limiter!(u, integrator, p, t)`.
+
+The argument `thread` determines whether internal broadcasting on
+appropriate CPU arrays should be serial (`thread = OrdinaryDiffEq.False()`,
+default) or use multiple threads (`thread = OrdinaryDiffEq.True()`) when
+Julia is started with multiple threads.
+
+## Reference
+@article{Alshina2008,
+  doi = {10.1134/s0965542508030068},
+  url = {https://doi.org/10.1134/s0965542508030068},
+  year = {2008},
+  month = mar,
+  publisher = {Pleiades Publishing Ltd},
+  volume = {48},
+  number = {3},
+  pages = {395--405},
+  author = {E. A. Alshina and E. M. Zaks and N. N. Kalitkin},
+  title = {Optimal first- to sixth-order accurate Runge-Kutta schemes},
+  journal = {Computational Mathematics and Mathematical Physics}
+}
+"""
+struct Alshina2{StageLimiter, StepLimiter, Thread} <: OrdinaryDiffEqAlgorithm
+  stage_limiter!::StageLimiter
+  step_limiter!::StepLimiter
+  thread::Thread
+end
+export Alshina2
```

Comment:
Since we already export this algorithm in the OrdinaryDiffEq.jl file, this export is redundant.